cgi-bin/web-watermarker.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
+# This script allows optional resizing of a picture, making text and image watermarks.
+
+import cgi
+import cgitb
+import html
+import os
+import sys
+
+from PIL import Image, ImageDraw, ImageEnhance, ImageFont
+from shutil import copyfile
+
+def resize_image(filename, width, height, outfolder):
+    """
+    Resizes the image to width x height box size and saves the
+    image to outfolder in PNG format.
+    :filename: path to the input image
+    :width: desired width of output image, an integer number
+    :height: desired height of output image, an integer number
+    :outfolder: path to the folder for the output image saving
+    """
+    image = Image.open(filename)
+    if image.mode != 'RGBA':
+        image = image.convert('RGBA')
+    image_width, image_height = image.size
+    if width <= image_width and height <= image_height:
```

Comment:
Yes, I have checked it.
It depends where we need a watermark). It's easy to make a watermark in the image corner instead of the image box.